app/controllers/shared/api.php

```diff
@@ -554,3 +554,11 @@
                 ->submit();
         }
     });
+
+App::init()
+    ->groups(['usage'])
+    ->action(function () {
+        if (App::getEnv('_APP_USAGE_STATS', 'enabled') != 'enabled') {
```

Comment:
```suggestion
        if (App::getEnv('_APP_USAGE_STATS', 'enabled') !== 'enabled') {
```
use strict inequality 👍

Reply:
Updated this.